src/neo4j_genai/schema.py

```diff
@@ -82,6 +89,51 @@ def get_schema(
     Returns:
         str: the graph schema information in a serialized format.
     """
+    structured_schema = get_structured_schema(driver)
```

Comment:
Are we planning to add more functions to get the schema as a text but using different formats? (I think I read it somewhere but I'm not sure). In this case, shall we try and find a more descriptive name for the `get_schema` function?

Reply:
You're probably thinking about [this card](https://trello.com/c/Bm4TVI5d) about enhancing the schema with values? But that would be used internally, so I don't think we have to worry about it mixing up the naming.

Reply:
I think this function could be used by LangChain/LLamaIndex eventually. But we can also think about that when/if needed in the future.